pkg/vgmanager/vgmanager_controller.go

```diff
@@ -318,12 +318,9 @@ func setStatus(status *lvmv1alpha1.VGStatus, instance *lvmv1alpha1.LVMVolumeGrou
 	found := false
 
 	vgStatuses := instance.Spec.LVMVGStatus
-	for _, vgStatus := range vgStatuses {
+	for i, vgStatus := range vgStatuses {
 		if vgStatus.Name == status.Name {
-			found = true
```

Comment:
- not toggling `found` may create a new vgstatus as per condition in line 331
- so `found` should still be set to `true` 🤔